packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -148,19 +152,22 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
             nonce,
             contractAddr: this.chainContract.address,
           })
-          const tx = await this.chainContract.appendQueueBatch(99999999, {
-            nonce,
-            gasPrice,
-          })
+          const txResponse = await this.chainContract.appendQueueBatch(
+            99999999,
```

Comment:
This 9999999 constant really needs a comment or some kind of explanation, it's unclear why this number exists here in the code.

Reply:
It exists to essentially append all pending queue batch elements, the assumption is that there will never be more than 9999999 pending queue elements at once. A potentially more clear way to do so would be to use `ethers.constants.MaxUint256`